lib/lutaml/xsd/element.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Lutaml
+  module Xsd
+    class Element < Lutaml::Model::Serializable
+      attribute :name, :string
+      attribute :type, :string
+      attribute :annotation, Annotation
+      attribute :unique, Unique
+
+      xml do
+        root "element", mixed: true
+        namespace "http://www.w3.org/2001/XMLSchema", "xsd"
+
+        map_attribute :name, to: :name
+        map_attribute :type, to: :type
+        map_element :annotation, to: :annotation, namespace: "http://www.w3.org/2001/XMLSchema", prefix: "xsd"
```

Comment:
I find it strange that we need to repeat the namespace in every single element. Instead, we should respect the default namespace of the element, and only override it with a nil prefix if we want no namespace?